fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_NavaidPage.js

```diff
@@ -48,19 +60,23 @@ class CDUNavaidPage {
         ];
 
         if (facility) {
-            template[2][0] = `{cyan}${WayPoint.formatIdentFromIcao(facility.databaseId)}{end}`;
+            let latLon = facility.location;
+            if (facility.sectionCode === 4 && facility.subSectionCode === 7) { // airport && ils
+                CDUNavaidPage.renderIls(facility, template);
+                latLon = facility.locLocation;
+            } else if (facility.sectionCode === 1 && facility.subSectionCode === 0) { // navaid && vhf
+                CDUNavaidPage.renderVorDme(facility, template);
+            }
+
+            template[2][0] = `{cyan}${facility.ident}{end}`;
 
+            // 3L
             template[3][0] = '\xa0\xa0\xa0\xa0LAT/LONG';
-            template[4][0] = `${new LatLong(facility.location.lat, facility.location.long).toShortDegreeString()}[color]green`;
+            template[4][0] = `{green}${new LatLong(latLon.lat, latLon.long).toShortDegreeString()}{end}`;
```

Comment:
I don't think `toShortDegreeString()` actually gives the correct format here. There should be a slash between lat and long.

Reply:
Good catch. Fixed on this and the waypoint page too.